addons/tailscale-relay/values.yaml

```diff
@@ -2,9 +2,9 @@
 replicas: 1
 
 image:
-  repository: docker.io/yosefmih/tailscale
+  repository: docker.io/mvisonneau/tailscale
   pullPolicy: IfNotPresent
-  # tag: <default to chart app version>
+  tag: v1.76.6
```

Comment:
This is quite a jump. Are we sure this is not breaking anything? I would prefer an incremental bump. Ik this repo doesn't put images often (their previous was 1.68), but I can build the versions between what we have and 1.76 if an upgrade is necessary.

Reply:
Tested this all over again using the same flow as when the PR was raised:
1. Deployed the older version of the chart(`1.72.1`) on a cluster with an existing web `Service`. The chart was configured to advertise the cluster's VPC as well as `Service` routes.
2. Confirmed I can access the web `Service` locally.
3. Upgraded the Tailscale chart on the cluster to `1.76.6`.
4. Confirmed I can still access the `Service` locally.

Good to merge this in now.